### PR TITLE
Portal release notes for 29-03-2022

### PR DESCRIPTION
**Pull request by sajumb**
Only one new feature for the portal in this release notes for the 29-03-2022. Please consider this today :-)

It has been decided that the changes that is Heartcore specifics will be added to a new Release notes section in the Our Heartcore folder that will be handled in another PR.

### CODE REVIEW

File: Umbraco-Cloud/Release-Notes/2022-03-29-ReleaseNotes.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Release Notes, March 29, 2022
+
+_Bandwidth Usage Top 10 for resources_
+
+### Key Takeaways
```
Comment (sofietoft) [inline Umbraco-Cloud/Release-Notes/2022-03-29-ReleaseNotes.md:5]:
```suggestion
## Key Takeaways
```

File: Umbraco-Cloud/Release-Notes/2022-03-29-ReleaseNotes.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- **Bandwidth Usage Top 10 for resources** - Wondering which resources of your Umbraco Cloud project that contributes with most bandwidth usage? Now you can find the answer in the new bandwidth Top 10 for resources on the project usage page.
+
+### Bandwidth Usage Top 10 for resources
```
Comment (sofietoft) [inline Umbraco-Cloud/Release-Notes/2022-03-29-ReleaseNotes.md:8]:
```suggestion
## Bandwidth Usage Top 10 for resources
```

**Review by sofietoft — CHANGES_REQUESTED** [review]

**Comment by sajumb** [comment]
Hi @sofietoft and @jonat123 
@hemraker wants to include this in the product update scheduled to tomorrow at 10 am. I believe the change request to be quite safe, so feel free to merge when possible. Thanks :-)

File: Umbraco-Cloud/Release-Notes/2022-03-29-ReleaseNotes.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+The bandwidth top 10 for referrers is now out of beta. And we are introducing a **new top 10 for resources** where you will see the resources that are contributing the most to the total bandwidth of your project. Each resource is represented by its path together with the number of requests and its total contribution of bandwidth.
+
+![Top10BandwidthPaths](https://user-images.githubusercontent.com/93588665/160573035-09f62ec3-9f4a-49c8-8e90-163c7418cb75.gif)
```
Comment (dampee) [inline Umbraco-Cloud/Release-Notes/2022-03-29-ReleaseNotes.md:11]: @sofietoft  do we allow images to be hosted externally?

Comment (jonat123) [reply Umbraco-Cloud/Release-Notes/2022-03-29-ReleaseNotes.md]: No images should add them to the images folder :) 

It has been resolved now :D

**Comment by jonat123** [comment]
@sajumb Looks good, merging in the PR :D

**Pull request merged**